pkg/option/option.go

```diff
@@ -80,7 +80,7 @@ type Options struct {
 	ClusterRequestTimeout string         `yaml:"cluster-request-timeout"`
 	Cluster               ClusterOptions `yaml:"cluster"`
 
-	// Path.
+	// FullMethod.
```

Comment:
please be careful when `Replace In All Files`, many code changes in this commit are wrong.

Reply:
yeah, I will be careful.
LOL, I used `rename(shift + F6)` instead of `replace`, but it looks like  `IntelliJ IDEA` has some problems judging references.